test/functional/test-batched-xhr.js

```diff
@@ -213,10 +213,8 @@ describes.sandboxed('BatchedXhr', {}, () => {
       return Promise.all([
         xhr.fetchText('/get?k=v1', {method: 'POST', body: {}}),
         xhr.fetchText('/get?k=v1', {method: 'POST', body: {}}),
-      ]).then(results => {
+      ]).then(() => {
         expect(fetchStub).to.be.calledTwice;
-        expect(results[0]).to.equal(TEST_RESPONSE);
-        expect(results[1]).to.equal(TEST_RESPONSE);
```

Comment:
But I notice you don't call `.then(res => res.text())` here, is that intentional? Is it not necessary for a POST? I believe checking the results matching the `TEST_RESPONSE` a second time for the POST method may catch unexpected behavior.